app/src/main/java/io/mochahub/powermeter/workoutsession/workoutsessiondialog/WorkoutRowModel.kt

```diff
@@ -0,0 +1,41 @@
+package io.mochahub.powermeter.workoutsession.workoutsessiondialog
+
+import android.widget.ArrayAdapter
+import android.widget.AutoCompleteTextView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import com.google.android.material.button.MaterialButton
+import io.mochahub.powermeter.R
+import io.mochahub.powermeter.models.Workout
+import io.mochahub.powermeter.shared.KotlinEpoxyHolder
+
+@EpoxyModelClass(layout = R.layout.row_workout_edit)
+abstract class WorkoutRowModel(
+    @EpoxyAttribute var workout: Workout,
+    @EpoxyAttribute var arrayAdapter: ArrayAdapter<String>,
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var addButtonClickListener: () -> Unit,
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash) var onExerciseSelected: (exercise: String) -> Unit
+) : EpoxyModelWithHolder<WorkoutRowModel.Holder>() {
+
+    override fun bind(holder: Holder) {
+
+        if (workout.exercise.name.isNotEmpty() || workout.exercise.name.isNotBlank()) {
```

Comment:
I think `.isNotBlank()` includes `.isNotEmpty()` if I'm reading [this](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-not-blank.html) correctly? 

> Returns true if this char sequence is not empty and contains some characters except of whitespace characters.

Reply:
Fixed here https://github.com/power-meter/android-client/pull/32/commits/f07ff66e35760e38b7af89e3ac54234a23caef43